ibm/mas_devops/roles/cp4d_db2wh_restore/README.md

```diff
@@ -0,0 +1,57 @@
+cp4d_db2wh_restore
+==========
+
+This role runs a restore procedure onto a CloudPak for Data DB2 Warehouse instance.
+In order to begin the restore process, you must have the required files to run a successful restore process in the chosen `DB2WH_BACKUP_DIR`:
+
+- DB2 backup files i.e `BLUDB.0.db2inst1.DBPART000.202XXXXXXXXXXX.001`
+- DB2 keystore files (.p12 and .sth)
+- DB2 instance master key label file (.kdb)
+
+Note: These files are generated automatically if you run `ibm.mas_devops.cp4d_db2wh_backup` role. If any of the above files are not found in the `DB2WH_BACKUP_DIR`, the restore process will fail.
+
+Role Variables
+--------------
+
+### db2wh_backup_dir
+Required. Local directory that stores the backup files to be used in the DB2 Warehouse restore process, i.e `/Users/Documents/db_backup`.
+
+- Environment Variable: `DB2WH_BACKUP_DIR`
+- Default: None
+
+### cpd_meta_namespace_target
```

Comment:
See earlier comments, I think we can remove this entirely and use the smart auto-detect namespace code that lives in cp4d_db2wh in this role so that the user doesn't have to worry about the cpd namespace input at all.

Reply:
fixed to be same as in `cp4d_db2wh` and `cp4d_db2wh_manage_hack` as you suggested.